pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -429,26 +429,28 @@ func (m *managerImpl) waitForPodsCleanup(podCleanedUpFunc PodCleanedUpFunc, pods
 }
 
 // reclaimNodeLevelResources attempts to reclaim node level resources.  returns true if thresholds were satisfied and no pod eviction is required.
-func (m *managerImpl) reclaimNodeLevelResources(resourceToReclaim v1.ResourceName, observations signalObservations) bool {
+func (m *managerImpl) reclaimNodeLevelResources(resourceToReclaim v1.ResourceName, capacityProvider CapacityProvider, pods []*v1.Pod) bool {
 	nodeReclaimFuncs := m.resourceToNodeReclaimFuncs[resourceToReclaim]
 	for _, nodeReclaimFunc := range nodeReclaimFuncs {
 		// attempt to reclaim the pressured resource.
-		reclaimed, err := nodeReclaimFunc()
+		nodeReclaimFunc()
```

Comment:
eating the error here.  i don't think we want to bail out but not sure if we want it to go by unreported.

Reply:
oh yeah, good catch.  i forgot to put error handling back in (i had moved it into the nodeReclaimFunc at one point).

Reply:
fixed